analytics.js

```diff
@@ -1105,6 +1130,14 @@ class Analytics {
         this.sendAdblockPageOptions = options.sendAdblockPageOptions;
       }
     }
+
+    // Session initialization
+    try {
```

Comment:
Move the exception handling into the session class.

Reply:
Done.